tests/support/mod.rs

```diff
@@ -112,10 +120,41 @@ impl Transform for MockTransform {
             }) => {
                 *val += self.increase;
             }
+            Event::Metric(Metric::AggregatedHistogram {
```

Comment:
@lukesteensen could we replace _ values with .. in this module?

Reply:
Yep! I believe you can use `..` to ignore the rest of the struct members.